VideoLocker/src/main/java/org/edx/mobile/discussion/DiscussionAPI.java

```diff
@@ -105,6 +105,9 @@ public TopicThreads searchThreadList(String courseId, String text, int pageSize,
     public ThreadComments getCommentList(String threadId, int pageSize, int page) throws RetroHttpException {
         return discussionService.getCommentList(threadId, pageSize, page);
     }
+    public ThreadComments getCommentListForQuestion(String threadId, int pageSize, int page, Boolean endorsed) throws RetroHttpException {
```

Comment:
> Methods should be separated by an empty line.

I thought I had added, but I guess it was missed.

>Any reason why endorsed is using the Boolean class type instead of primitive? Similarly, flagged, voted, and following in the succeeding methods.

I remember this as a mistake during testing, don't know why others are not primitive.